src/Sil/Bundle/ContactBundle/Resources/public/js/show/contact.js

```diff
@@ -0,0 +1,346 @@
+
+        var data = new FormData($(this).get(0));
+
+        $.ajax({
+            url: url,
+            method: 'POST',
+            data: data,
+            processData: false,
+            contentType: false,
+            success: response => {
+                $('#address-list').replaceWith($(response));
+
+                $('#address-modal').modal('hide');
+                hideAddressLoader();
+            }
+        });
+
+        return false;
+    })
+
+    /**
+     * Close address form
+     */
+    .on('click', '#cancel-address', () => {
+        $('#address-modal').modal('hide');
+    })
+
+    /**
+     * Edit address
+     */
+    .on('click', '#address-list .edit-address', function() {
+        showAddressLoader();
+
+        var address = $(this).parents('.contact-address');
+
+        $.get(address.data('edit-url'), response => {
+            $('#address-form').parent().replaceWith($(response));
+            $('#address-form').data('edit', true);
+            $('#address-modal').modal('show');
+
+            initDropdowns();
+            hideAddressLoader();
+        });
+
+        return false;
+    })
+
+    /**
+     * Remove address from list
+     */
+    .on('click', '#address-list .remove-address', function() {
+        showAddressLoader();
+        var address = $(this).parents('.contact-address');
+
+        $.get(address.data('remove-url'), response => {
+            $('#address-list').replaceWith($(response));
+            hideAddressLoader();
+        });
+
+        return false;
+    })
+
+    /**
+     * Set address as contact default
+     */
+    .on('click', '#address-list .make-default', function() {
+        showAddressLoader();
+        var contactId = $('#contact').data('id');
+        var address = $(this).parents('.contact-address');
+
+        $.get(address.data('make-default-url'), response => {
+            $('#address-list').replaceWith($(response));
+            hideAddressLoader();
+        });
+    })
+
+    /**
+     * Remove a note from the list
+     */
+    .on('click', '.remove-note', function() {
+        $(this).parents('.contact-note').remove();
+    })
+
+;
+
+var showContactLoader = function() {
+    $('#contact-loader').addClass('active');
+}
+
+var hideContactLoader = function() {
+    $('#contact-loader').removeClass('active');
+}
+
+var showGroupLoader = function() {
+    $('#group-loader').addClass('active');
+}
+
+var hideGroupLoader = function() {
+    $('#group-loader').removeClass('active');
+}
+
+var showPhoneLoader = function() {
+    $('#phone-list-loader').addClass('active');
+}
+
+var hidePhoneLoader = function() {
+    $('#phone-list-loader').removeClass('active');
+}
+
+var showAddressLoader = function() {
+    $('#address-list-loader').addClass('active');
+}
+
+var hideAddressLoader = function() {
+    $('#address-list-loader').removeClass('active');
+}
+
+var initDropdowns = function() {
+    $('.ui.dropdown').dropdown();
+}
+
+var resetForm = function(form) {
```

Comment:
You can use attribute `autocomplete="off"` to avoid resetting manually the form.